src/app/data/data-types.ts

```diff
@@ -1,4 +1,4 @@
-export interface Character {
+export interface CharacterTest {
```

Comment:
i see this interface used to set up the mock class but i'm wondering if we can just use the Character class and make several properties optional. Unless you're planning on using this class for something else?

Reply:
I'm planning on just replacing it with character in my next CL.